mautic/templates/deployment.yaml

```diff
@@ -54,6 +54,10 @@ spec:
               value: {{ .Values.runCronJobs | quote }}
             - name: MAUTIC_TRUSTED_PROXIES
               value: {{ .Values.trustedProxies | quote }}
+          lifecycle:
+            postStart:
+              exec:            
+                command: ["/bin/sh", "-c", "php /var/www/html/bin/console cache:warmup && chown www-data.www-data -R /var/www/html/var"]
```

Comment:
Why not `sudo -u www-data php /var/www/html/bin/console cache:warmup`?

Reply:
I have added "sudo -u www-data"

Reply:
So remove the "&& chown www-data.www-data -R /var/www/html/var"

Reply:
So remove the "&& chown www-data.www-data -R /var/www/html/var"